add_car.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+require_once "vendor/autoload.php";
+require_once "src/utils.php";
+
+use CarStore\CarModel;
+use CarStore\Car;
+
+if (
+    isset($_POST["name"]) &&
+    isset($_POST["year_made"]) &&
+    isset($_POST["zero_sixty"]) &&
+    isset($_POST["price"]) &&
+    isset($_POST["brand"])
+) {
+    $name = $_POST["name"];
+    $year_made = $_POST["year_made"];
+    $zero_sixty = $_POST["zero_sixty"];
+    $price = $_POST["price"];
+    $brand = $_POST["brand"];
+
+    $car = new Car($name, $year_made, $zero_sixty, $price, $brand);
+
+    $validate = validateDataFields($car);
+
+    if ($validate == false) {
+        return;
```

Comment:
using return with no value is bad practice, it's also not in a function so doesn't really work as expected